framework/framework.md

```diff
@@ -24,6 +24,39 @@ The following terms are used in this document with a specific meaning. Definitio
 * **target equipment**: [[IEC]](https://std.iec.ch/terms/terms.nsf/0/B9C576C31B4E488DC125771F002F58E9?OpenDocument) the equipment, machinery, apparatus or plant for which *safety* is to be achieved
 * **target system**: The *safety-related system* (specific or assumed) under consideration. For the purposes of this document, this is assumed to include a *component* incorporating software based on *Linux*
 
+### ISO 26262 Definitions
+
+* **architecture**: The representation of the structure of the *system* that allows identification of building blocks, their boundaries and interfaces, and includes the allocation of requirements to those building blocks.
```

Comment:
I feel this should be 'system architecture' - we might also have architecture for an element or component